.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -34,4 +34,5 @@
 ⏪ Reverting a previous change
 ♻️ Refactoring
 🚮 Deleting code
+🔬 Experimental code
```

Comment:
The microscope icon is skeuomorphic on some platforms.

https://emojipedia.org/microscope/

If that's a problem, a couple alternatives:

https://emojipedia.org/test-tube/ (this too works nicely)
https://emojipedia.org/alembic/ (meh)

Reply:
Changed to 🧪. It was my first choice but I wasn't sure if it looks good :)